test/ecto/changeset_test.exs

```diff
@@ -2,6 +2,21 @@ defmodule Ecto.ChangesetTest do
   use ExUnit.Case, async: true
   import Ecto.Changeset
 
+  defmodule Permalink do
```

Comment:
Can we use the `Comment` model? So we don't add new models to the suite? Thank you. :)